packages/components/src/components/chip/chip.scss

```diff
@@ -53,14 +60,13 @@
 
 :host([appearance="outline"]),
 :host([appearance="outline-fill"]) {
+  --calcite-internal-chip-close-icon-color: var(--calcite-color-text-3);
+  --calcite-internal-chip-close-icon-color-hover: var(--calcite-color-text-3);
```

Comment:
I don't think we need to change the icon color on hover here. We should be inheriting Action's hover and press icon colors.

Reply:
It doesn't look right inheriting Action's default hover and press icon colors.  That's why I need to set `--calcite-action-text-color-press` like I'm doing down below.

Reply:
You can ignore these comments.  After the video you sent I realized I just needed to remove these 2 lines to let Action's default styling take over for outline and outline-fill.  Thanks for the catch!